### DISCUSSION TITLE
The assets placed in Github doesn't actually match with the assets placed inside game (specifically in APK Editor)

### DISCUSSION THREAD

**Discussion by NeptuneNeptix** [luckydog7/Funkin-android#1453] [Q&A]
I've been questioning why is the open source by luckydog posted on Github doesn't actually match places with real inside game assets places (specifically APK Editor)
This (might) make some troubles while making a mod on mobile or just simply reskin stuff.
Example: your going to make a FNF but bad mod of yourself version, you looked at this github open source, take stuff from it, redraw and put it back in the game. (Using APK editor ofc)
But then you realized you can't find the thing you want to replace (example: spooky kids assets)

Also bad english beware of them!!1!1!!1!

**Comment by SKYLR460** [comment]
U can replace the game files skins and it works it doesnt need the source file

**Comment by POKO-dev** [comment]
> I've been questioning why is the open source by luckydog posted on Github doesn't actually match places with real inside game assets places (specifically APK Editor)
> This (might) make some troubles while making a mod on mobile or just simply reskin stuff.
> Example: your going to make a FNF but bad mod of yourself version, you looked at this github open source, take stuff from it, redraw and put it back in the game. (Using APK editor ofc)
> But then you realized you can't find the thing you want to replace (example: spooky kids assets)
> 
> Also bad english beware of them!!1!1!!1!
Bruuuh noob programs

**Comment by POKO-dev** [comment]
> I've been questioning why is the open source by luckydog posted on Github doesn't actually match places with real inside game assets places (specifically APK Editor)
> This (might) make some troubles while making a mod on mobile or just simply reskin stuff.
> Example: your going to make a FNF but bad mod of yourself version, you looked at this github open source, take stuff from it, redraw and put it back in the game. (Using APK editor ofc)
> But then you realized you can't find the thing you want to replace (example: spooky kids assets)
> 
> Also bad english beware of them!!1!1!!1!

Need the the fla and the source hx and note the fnf file

**Comment by POKO-dev** [comment]
> I've been questioning why is the open source by luckydog posted on Github doesn't actually match places with real inside game assets places (specifically APK Editor)
> This (might) make some troubles while making a mod on mobile or just simply reskin stuff.
> Example: your going to make a FNF but bad mod of yourself version, you looked at this github open source, take stuff from it, redraw and put it back in the game. (Using APK editor ofc)
> But then you realized you can't find the thing you want to replace (example: spooky kids assets)
> 
> Also bad english beware of them!!1!1!!1!

FACK YOU PUT THE GAME FILE IN GITHUB THAT'S NOTE OPEN SOURCE YOU NEED THE HX AND FLA FILE

**Reply by NeptuneNeptix** [reply]
> > I've been questioning why is the open source by luckydog posted on Github doesn't actually match places with real inside game assets places (specifically APK Editor)
> > This (might) make some troubles while making a mod on mobile or just simply reskin stuff.
> > Example: your going to make a FNF but bad mod of yourself version, you looked at this github open source, take stuff from it, redraw and put it back in the game. (Using APK editor ofc)
> > But then you realized you can't find the thing you want to replace (example: spooky kids assets)
> > Also bad english beware of them!!1!1!!1!
> 
> FACK YOU PUT THE GAME FILE IN GITHUB THAT'S NOTE OPEN SOURCE YOU NEED THE HX AND FLA FILE

Damn you have a very bad english I cannot understand any gibberish you say.

**Reply by POKO-dev** [reply]
> > > I've been questioning why is the open source by luckydog posted on Github doesn't actually match places with real inside game assets places (specifically APK Editor)
> > > This (might) make some troubles while making a mod on mobile or just simply reskin stuff.
> > > Example: your going to make a FNF but bad mod of yourself version, you looked at this github open source, take stuff from it, redraw and put it back in the game. (Using APK editor ofc)
> > > But then you realized you can't find the thing you want to replace (example: spooky kids assets)
> > > Also bad english beware of them!!1!1!!1!
> > 
> > 
> > FACK YOU PUT THE GAME FILE IN GITHUB THAT'S NOTE OPEN SOURCE YOU NEED THE HX AND FLA FILE
> 
> Damn you have a very bad english I cannot understand any gibberish you say.

Look you can't you can't open the source because you don't use the  FACKING HAXE FLIXEL

**Reply by NeptuneNeptix** [reply]
> > > > I've been questioning why is the open source by luckydog posted on Github doesn't actually match places with real inside game assets places (specifically APK Editor)
> > > > This (might) make some troubles while making a mod on mobile or just simply reskin stuff.
> > > > Example: your going to make a FNF but bad mod of yourself version, you looked at this github open source, take stuff from it, redraw and put it back in the game. (Using APK editor ofc)
> > > > But then you realized you can't find the thing you want to replace (example: spooky kids assets)
> > > > Also bad english beware of them!!1!1!!1!
> > > 
> > > 
> > > FACK YOU PUT THE GAME FILE IN GITHUB THAT'S NOTE OPEN SOURCE YOU NEED THE HX AND FLA FILE
> > 
> > 
> > Damn you have a very bad english I cannot understand any gibberish you say.
> 
> Look you can't you can't open the source because you don't use the FACKING HAXE FLIXEL

do you even read the main discussion

**Comment by blazingcody** [comment]
I tried to put a little man garcello mod over boyfriend at a part of the song it freezes

**Comment by blazingcody** [comment]
https://gamebanana.com/wips/55063
